Hardware/Hardware/CardReader/StripeCardReader/StripeCardReaderService.swift

```diff
@@ -375,6 +375,17 @@ extension StripeCardReaderService: CardReaderService {
 
 // MARK: - Payment collection
 private extension StripeCardReaderService {
+    /// Returns the id of the connected reader, if any
+    ///
+    func readerIDForIntent() -> String? {
+        let connectedReaders = connectedReadersSubject.value
+        guard connectedReaders.count == 1 else {
+            return nil
+        }
+
+        return connectedReaders[0].id
```

Comment:
nit: I prefer safer access where it's possible, just as a habit (this case has already safety check in guard)
```suggestion
        return connectedReaders.first?.id
```

Reply:
Good idea. Will do

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/4691/commits/8264301722f71d678d90b84913e368c0a1c375e2